InfiniteBlanK3T.Calculator/Program.cs

```diff
@@ -0,0 +1,169 @@
+﻿namespace CalculatorProgram
```

Comment:
you can use this notation to have one less indentation level and hence save space :)

![image](https://user-images.githubusercontent.com/108711476/211977571-2e4da192-5a39-40ac-979e-aa2e0e819604.png)